### PR TITLE
[POWRPROF_APITEST] Fix 2+1 Cppcheck reports

### PR DESCRIPTION
**Pull request by SergeGautherie**
Note that both tests are currently skipped...

### CODE REVIEW

File: modules/rostests/apitests/powrprof/power.c [hunk 1]
```diff
@@ -4620,7 +4620,8 @@ pPP_original->user.OptimizeForPowerDc=1;
    ret = ValidatePowerPolicies(&gpp,&pp);
    ok(ret, "function was expected to succeed error %i\n",(UINT)GetLastError());
    ok(globalcompare(gpp,gpp_compare),"Difference Found\n");
-   pp_compare.user.MaxSleepAc=4;
+   // FIXME: Duplicate line.
+   // pp_compare.user.MaxSleepAc=4;
```
Comment (binarymaster) [inline modules/rostests/apitests/powrprof/power.c:4624]: Considering the code above where variable is initialized, you can safely delete this line.

Comment (SergeGautherie) [reply modules/rostests/apitests/powrprof/power.c]: Deleted.

File: modules/rostests/apitests/powrprof/power.c [hunk 2]
```diff
@@ -60,7 +60,7 @@ void test_CallNtPowerInformation(void)
    /* LastSleepTime tests */
    retval = CallNtPowerInformation(LastSleepTime, 0, 0, 0, 0);
    ok(retval == STATUS_SUCCESS, "function result wrong expected STATUS_SUCCESS but got %d\n", (UINT)retval);
-   retval = CallNtPowerInformation(LastSleepTime, &atime, sizeof(sizeof(ULONGLONG)), 0, 0);
+   retval = CallNtPowerInformation(LastSleepTime, &atime, sizeof(atime), 0, 0);
```
Comment (ThFabba) [inline modules/rostests/apitests/powrprof/power.c:63]: I agree that `sizeof(variable)` is generally preferable to `sizeof(TYPE)` but it's still a style choice, and the #1 rule of style is to be consistent, so please use `sizeof(ULONGLONG)`. Then this becomes a trivial fix.

Comment (SergeGautherie) [reply modules/rostests/apitests/powrprof/power.c]: Reverted.

**Pull request merged**